src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,258 @@
+            numberOfTasks = numberOfTasks;
+            return numberOfTasks;
+        }
+        newTask = new Deadline(splitTaskDescriptionArray[0], splitTaskDescriptionArray[1]);
+        numberOfTasks = addToList(newTask, taskList, numberOfTasks);
+        return numberOfTasks;
+    }
+
+
+    private static int addTodoToList(Task[] taskList, int numberOfTasks, String userInput) {
+        Task newTask;
+        String[] splitTaskDescriptionArray = splitTaskDescription(userInput);
+        if (isDescriptionBlank(splitTaskDescriptionArray)) {
+            displayNoDescriptionError();
+            numberOfTasks = numberOfTasks;
+            return numberOfTasks;
+        }
+        newTask = new Todo(splitTaskDescriptionArray[0], splitTaskDescriptionArray[1]);
+        numberOfTasks = addToList(newTask, taskList, numberOfTasks);
+        return numberOfTasks;
+    }
+
+    private static int addToList(Task newTask, Task[] taskList, int numberOfTasks) {
+        taskList[numberOfTasks] = newTask;
+        numberOfTasks = numberOfTasks + 1;
+        System.out.println("Got it. I've added this task: " + newTask.toString());
+        return numberOfTasks;
+
+    }
+
+    private static void displayNoDescriptionError() {
+        System.out.println("Sorry! This task lack description!");
+    }
+
+    private static void displayGoodbye() {
+        System.out.println(System.lineSeparator());
+        System.out.println("Bye. Hope to see you again soon! Maybe next time");
+    }
+
+    private static void displayHello() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke.Duke\nWhat can I do for you?");
+    }
+
+    private static String[] splitTaskDescription(String input) {
+        String[] returnSplit = new String[3];
+        if (!input.contains("/")) {
+            String[] obtainedDescription = input.split(" ", 2);
+            if (obtainedDescription.length == 1 || obtainedDescription[1].isBlank()) {
+                returnSplit[0] = "";
+            } else {
+                returnSplit[0] = obtainedDescription[1].trim();
+            }
+            returnSplit[1] = "";
+            returnSplit[2] = "";
+            return returnSplit;
+        }
+        String[] obtainedSplit = input.split("/");
+        String[] obtainedDescription = obtainedSplit[0].split(" ", 2);
+        if (obtainedDescription.length == 1 || obtainedDescription[1].isBlank()) {
+            returnSplit[0] = "";
+            returnSplit[1] = "";
+        } else {
+            returnSplit[0] = obtainedDescription[1].trim();
+            returnSplit[1] = obtainedSplit[1].trim();
+            returnSplit[2] = "yes";
+        }
+        return returnSplit;
+    }
+

```

Comment:
Might consider to implement the 0/1/2... in returnSplit[0]  returnSplit[1] to constants that the names suggests the specific info it contains.

Reply:
Hmm I think I might create constants for the 0/1/2 directly inside this method. Thanks!